Tests/LibWeb/Layout/input/iframe-append-content-of-document-body-crash.html

```diff
@@ -0,0 +1,11 @@
+<iframe id=iframe></iframe>
+<select>Text Node
+    <option id=option></option>
+</select>
+
+<script>
+    document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Huh, this test passes individually but seems to cause problems with other tests. I suppose it might be because load event is never fired for the document. In this test 🤔 Maybe it's not possible to test this at all with our current infrastructure

Reply:
I've ended up removing the test, it is covered by WPT, but I don't think because of how weird this test is that it is actually testable with the Text/Layout test infrastructure